client/src/Components/CommonComponent/Navbar/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './style.css';
+
+const Navbar = () => (
+  <div className="navbar">
+    <div className="controlBoard">لوحة التحكم</div>
+    <div className="circle">D</div>
```

Comment:
'D' is the first letter of the user ,so change it from static and add it as props